api/v1/curvecluster_types.go

```diff
@@ -236,15 +231,15 @@ type SnapShotCloneSpec struct {
 	S3Config S3ConfigSpec `json:"s3Config,omitempty"`
 }
 
-// S3Config
+// S3ConfigSpec ...
 type S3ConfigSpec struct {
 	AK                 string `json:"ak,omitempty"`
 	SK                 string `json:"sk,omitempty"`
 	NosAddress         string `json:"nosAddress,omitempty"`
 	SnapShotBucketName string `json:"bucketName,omitempty"`
 }
 
-// StorageScopeSpec
+// StorageScopeSpec ...
```

Comment:
So, maybe you can add more description for EtcdSpec, MdsSpec, SnapShotCloneSpec, S3ConfigSpec.

Reply:
PTAL